index.html

```diff
@@ -18,52 +18,259 @@
+              <li class="list-item year">2015</li>
+            </ul>
+            <p class="text_paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <div class="btn-container-flex">
+              <button type="button" class="btn-sm">html</button>
+              <button type="button" class="btn-md">css</button>
+              <button type="button" class="btn-lg">javaScript</button>
+            </div>
+            <div class="btn-container">
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
+      <h2 class="about-title">About<br />Myself<br /></h2>
+      <p class="paragraph">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+
+      <span class="connect">LETS CONNECT</span>
+
+      <div class="socialmedia_icons_container">
+        <ul class="media_icons">
+          <li>
+            <a href="#">
+              <img class="media--images" src="images/icons/twitter.svg" alt="Twitter Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img class="media--images" src="images/icons/linkedin.png" alt="LinkedIn Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img class="media--images" src="images/icons/m.svg" alt="Media Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/cat.svg" alt="Github Icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="images/icons/fingers.svg" alt="Victory Icon" />
+            </a>
+          </li>
+        </ul>
+        <div class="btn-container about-btn">
+          <button type="button" class="btn-xxl btn-resume">
+            Get my resume
+          </button>
+        </div>
+      </div>
+
+      <ul class="accordion">
+        <li>
+          <div class="accordion-container">
+            <a class="skill--description-link" href="#"
+              ><span class="skill--description">Languages</span></a>
+            <ion-icon
+              class="chevron-icons"
+              name="chevron-down-sharp"></ion-icon>
+          </div>
+          <ul>
+            <li class="skill">
+              <img class="skill-image" src="images/works/skill_1.png" alt="" />
+              <span class="skill-name">Javascript</span>
             </li>
-            <li>
-              <a href="#">
-                <img src="images/icons/cat.svg" alt="Github Icon" />
-              </a>
+            <li class="skill">
```

Comment:
- [OPTIONAL]  Since this project focuses on your `Works Section` only - I would recommend adding something like 🤓 
```css
    margin: 6px 36px 12px 36px;
```

Reply:
@blessedjasonmwanza Thank you so much for this. my project looks so much better now!!